### ISSUE TITLE
Add tooltip for type-ahead interaction

### ISSUE THREAD

**Issue by sehyod** [refstudio/refstudio#271]

**Comment by hammer** [comment]
How will the user request a completion? What context will be sent to the language model? How will we display suggested completions?

To ensure these issues are linked, I'm mentioning here that we had a lot of discussion at https://github.com/refstudio/refstudio/issues/78 on tooltip design for rewrites.

**Comment by sehyod** [comment]
I don't think we have any clear answers to these questions at the moment. The idea would be to start implementing a tooltip that opens with a shortcut so we can start playing around with it to get a better sense of how to improve it. Perhaps we could update this issue with screenshots as we work on the component, so we can discuss it.

As for what we need to send to the language model, we need to sync with @gjreda, but the frontend basically has access to all the editor content and can send anything we think would be useful  (current sentence, current block, ...)
